plugin/al/plugin/AL_USDMayaTestPlugin/py/testUSDMayaPythonModuleOnly.py

```diff
@@ -0,0 +1,48 @@
+'''Test that basic functionality of the python module works without maya.
```

Comment:
Missing license note - see https://github.com/Autodesk/maya-usd/blob/dev/test/lib/ufe/ufeTestPlugins/ufeTestCmdsPlugin.py#L4

Reply:
Will do...

Reply:
Done - and also added the copyright to other py tests in the dir!